packages/nextjs-template/utils/env.js

```diff
@@ -0,0 +1,37 @@
+const { z } = require("zod");
+const path = require("path");
+const { FRONTEND_CONSTANTS } = require("@dendronhq/common-frontend");
+const { parse } = require("@dendronhq/common-all");
+
+const serverSchema = z.object({
+  DATA_DIR: z
+    .string()
+    .default(path.join(process.cwd(), FRONTEND_CONSTANTS.DEFAULT_DATA_DIR)),
+  PUBLIC_DIR: z.string().default(path.join(process.cwd(), "public")),
```

Comment:
this will fail if we run next in a different directory. replace `process.cwd` with `path.join(__dirname, "..")`

Reply:
[`fff9492` (#3542)](https://github.com/dendronhq/dendron/pull/3542/commits/fff94929d74a67f96f076a79f6666793edc26e93)